buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -96,33 +124,19 @@ class AirbytePythonPlugin implements Plugin<Project> {
             pip 'coverage[toml]:6.3.1'
         }
 
-
-        project.task('isortFormat', type: PythonTask) {
-            module = "isort"
-            command = "--settings-file=${project.rootProject.file('pyproject.toml').absolutePath} ./"
-        }
-
         project.task('isortReport', type: PythonTask) {
             module = "isort"
             command = "--settings-file=${project.rootProject.file('pyproject.toml').absolutePath} --diff --quiet ./"
             outputPrefix = ''
         }
 
-        project.task('blackFormat', type: PythonTask) {
-            module = "black"
-            // the line length should match .isort.cfg
-            command = "--config ${project.rootProject.file('pyproject.toml').absolutePath} ./"
-            dependsOn project.rootProject.licenseFormat
-            dependsOn project.isortFormat
-        }
-
```

Comment:
@postamar - For my info, post merge, what’s the correct/best entrypoint for python (or generic) linting and auto-formatting?

Reply:
`:format` from the repo root